system/views/renderers/asset/svg/svgHtml.cfm

```diff
@@ -0,0 +1,9 @@
+<cfscript>
+    assetBinary = args.assetBinary ?: "";
+</cfscript>
+
+<cfoutput>
+    <cfif !isEmpty( assetBinary ) >
+        #decodeForHTML( assetBinary )#
```

Comment:
Lucee built-in functions should use headed camel case. e.g. IsEmpty() and DecodeForHTML()